Makefile

```diff
@@ -32,31 +32,46 @@ endif
 MSBUILD_ARGS := /nologo '/consoleloggerparameters:Verbosity=minimal;summary' /p:Configuration=$(BUILD_CONFIGURATION)
 
 ifneq ($(BUILD_LOG_PATH),)
-	MSBUILD_ARGS += /filelogger '/fileloggerparameters:Verbosity=normal;logFile=$(BUILD_LOG_PATH)'
+    MSBUILD_ARGS += /filelogger '/fileloggerparameters:Verbosity=normal;logFile=$(BUILD_LOG_PATH)'
 endif
 
+MSBUILD_MAIN_ARGS := $(MSBUILD_ARGS)
+MSBUILD_BOOTSTRAP_ARGS := $(MSBUILD_ARGS)
```

Comment:
It's kind of confusing to put `dotnet build` args in a list called `MSBUILD_BOOTSTRAP_ARGS`